RecoParticleFlow/PFSimProducer/plugins/SimPFProducer.cc

```diff
@@ -318,12 +318,26 @@ void SimPFProducer::produce(edm::StreamID, edm::Event& evt, const edm::EventSetu
 	    if( supercluster_index != -1 ) {
 	      edm::Ref<reco::PFBlockCollection> blockRef(blocksHandle,block_index);
 	      for( const auto& elem : blockRef->elements() ) {
-		auto ref = elem.clusterRef();
+		const auto& ref = elem.clusterRef();
 		if( !usedSimCluster[ref.key()] ) {
 		  candidate.addElementInBlock(blockRef,elem.index());
 		  usedSimCluster[ref.key()] = true;
 		}
 	      }
+	      
+              //*TODO* cluster time is not reliable at the moment, so just keep time from the track if available
+              if (false) {
```

Comment:
a macro is a better handle if this code needs to be enabled for some private tests and it is too taxing to introduce a config parameter

Reply:
I mean ```#ifdef ENABLE_CLUSTER_TIME```

Reply:
To be honest I could simply remove these in a follow-up pull request or additional commit, and then we'll be able to keep track of them anyways through the commit history.

I would vote for follow-up pull request in case there's no other reason to retrigger tests.  (And then I can add the commit to the 93x backport in parallel)